app/src/main/java/com/main/mealplanner/dto/MealPlan.kt

```diff
@@ -9,7 +9,11 @@ import java.time.LocalDateTime
 import java.util.*
 
 @Entity(tableName="mealplan")
-data class MealPlan(var CookSchedule: LocalDateTime, var RecipeId : String, var OwnerEmail : String, @PrimaryKey @SerializedName("id") var MealPlanId:Int = 0) {
+data class MealPlan(
+        var CookSchedule: LocalDateTime
+        , var RecipeId : String
+        , var OwnerEmail : String
+        , @PrimaryKey @SerializedName("id") var MealPlanId:Int = 0) {
```

Comment:
This bias, for me I having to scroll across to read what the variable are, this way it easier for me.